trino-s3-proxy/src/main/java/io/trino/s3/proxy/server/rest/TrinoS3ProxyRestConstants.java

```diff
@@ -17,7 +17,7 @@ public final class TrinoS3ProxyRestConstants
 {
     private TrinoS3ProxyRestConstants() {}
 
-    public static final String BASE_PATH = "/api/v1/s3Proxy/";
-    public static final String S3_PATH = BASE_PATH + "s3";
-    public static final String STS_PATH = BASE_PATH + "sts";
+    private static final String BASE_PATH = "/api/v1/s3Proxy/";
+    public static final String DEFAULT_S3_PATH = BASE_PATH + "s3";
```

Comment:
These constants aren't really useful any more. I'd inline them. Let's make `BASE_PATH` a config as well.

Reply:
The idea was that `S3_PATH` and `STS_PATH` are now fully independent. `BASE_PATH` is only used here for the default.

So someone may have `s3proxy.s3.prefix=/s3` and `s3proxy.sts.prefix=/sts` and there's no common part to them

Agreed on removing the constants